specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -0,0 +1,157 @@
+{
+    "swagger": "2.0",
```

Comment:
>swagger [](start = 5, length = 7)

I noticed that there's an existing swagger for api version 2018-03-01-preview that includes resources like enrollmentAccounts, billingPeriods, etc.

Are you planning to have different API versions for some resource vs others. Or will you copy all the existing resources into this API version?

Reply:
Also, there is already a swagger for version 2018-11-01-preview in that location.

---
In reply to: [257442407](https://github.com/Azure/azure-rest-api-specs/pull/5209#discussion_r257442407) [](ancestors = 257442407)

Reply:
Made the changes